tests/test_auth.py

```diff
@@ -132,7 +132,7 @@ def client_credential_oauth2credential():
         access_token=ACCESS_TOKEN,
         expires_in_seconds=EXPIRES_IN_SECONDS,
         scopes=CLIENT_CREDENTIALS_SCOPES,
-        grant_type=auth.CLIENT_CREDENTIAL_GRANT,
+        grant_type=auth.CLIENT_CREDENTIALS_GRANT,
```

Comment:
nice catch

Reply:
will this break existing implementations?

Reply:
I don't think anyone uses CLIENT_CREDENTIAL_GRANT externally as all the endpoints previously required authorization code grant. Either way I will bump to a major version with to indicate bc change.